code/game/objects/items/devices/scanners.dm

```diff
@@ -495,3 +496,62 @@ Subject's pulse: ??? BPM"})
 	icon_state = "adv_spectrometer"
 	details = 1
 	origin_tech = Tc_MAGNETS + "=4;" + Tc_BIOTECH + "=2"
+
+/obj/item/device/breathalyzer
+	name = "breathalyzer"
+	icon_state = "health"
+	item_state = "analyzer"
+	desc = "A hand-held scanner that is able to determine the amount of ethanol in the breath of the subject."
+	flags = FPRINT
+	siemens_coefficient = 1
+	slot_flags = SLOT_BELT
+	throwforce = 3
+	w_class = W_CLASS_TINY
+	throw_speed = 5
+	starting_materials = list(MAT_IRON = 50)
+	w_type = RECYK_ELECTRONIC
+	melt_temperature = MELTPOINT_PLASTIC
+	origin_tech = Tc_ENGINEERING + "=1;" + Tc_BIOTECH + "=1"
+
+	var/legal_limit
+
+/obj/item/device/breathalyzer/New()
+	var/datum/reagent/ethanol/E = new
+	legal_limit = E.slur_start //inb4 shitcurity arrests people for being over the legal limit
```

Comment:
Why do you initiate this in `New()` ? You could do it directly in the declaration.
Additonaly, if you need to keep that format, use `initial(E.slur_start)` to avoid having to instantiate the reagent.

Reply:
Doing it in the declaration complained about it having to be constant. Was I doing something wrong?

Reply:
Use `initial` but keep it in New, you can't have `E` in the declaration. (well, you could, but let's not)